biliup/uploader.py

```diff
@@ -36,6 +37,10 @@ def upload(data):
         if threshold:
             data['threshold'] = threshold
 
+        # 过滤自定义封面
+        if context.get('cover_path') == "room_cover":
```

Comment:
特殊变量用花括号和{title}等统一，是不是好一点
```suggestion
        if context.get('cover_path') == "{room_cover}":
```